### ISSUE TITLE
2.0.0-alpha.73 Markdown Frontmatter can't handle empty description

### ISSUE THREAD

**Issue by TomPeirs** [facebook/docusaurus#4652]
## 🐛 Bug Report

(A clear and concise description of what the bug is)

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

(Write your answer here.)

## To Reproduce

1. I updated to the latest version of docusaurus
1. NPM run start
1. Have a simple MD file with an empty description frontmatter: 
![image](https://user-images.githubusercontent.com/68061801/115472720-bb5a6b00-a23a-11eb-9f2c-1a7733a9c889.png)

## Expected behavior

Build and compile without an issue. (was working in the previous version)

## Actual Behavior

```
ValidationError: "description" must be a string
    at Object.exports.process (D:\Prototype\alpha72\node_modules\joi\lib\errors.js:184:16)
    at Object.internals.entry (D:\Prototype\alpha72\node_modules\joi\lib\validator.js:150:26)
    at Object.exports.entry (D:\Prototype\alpha72\node_modules\joi\lib\validator.js:27:30)
    at internals.Base.validate (D:\Prototype\alpha72\node_modules\joi\lib\base.js:548:26)
    at Object.internals.assert (D:\Prototype\alpha72\node_modules\joi\lib\index.js:225:27)
    at Object.attempt (D:\Prototype\alpha72\node_modules\joi\lib\index.js:107:26)
    at Object.assertDocFrontMatter (D:\Prototype\alpha72\node_modules\@docusaurus\plugin-content-docs\lib\docFrontMatter.js:22:28)
    at Object.processDocMetadata (D:\Prototype\alpha72\node_modules\@docusaurus\plugin-content-docs\lib\docs.js:66:22) 
    at processVersionDoc (D:\Prototype\alpha72\node_modules\@docusaurus\plugin-content-docs\lib\index.js:89:35)        
    at Array.map (<anonymous>)
    at loadVersionDocsBase (D:\Prototype\alpha72\node_modules\@docusaurus\plugin-content-docs\lib\index.js:96:45)      
    at async loadVersion (D:\Prototype\alpha72\node_modules\@docusaurus\plugin-content-docs\lib\index.js:100:34)       
    at async Promise.all (index 0)
    at async Object.loadContent (D:\Prototype\alpha72\node_modules\@docusaurus\plugin-content-docs\lib\index.js:167:33)    at async D:\Prototype\alpha72\node_modules\@docusaurus\core\lib\server\plugins\index.js:53:46
    at async Promise.all (index 0)

```

## Your Environment
- Docusaurus version used:  2.0.0-alpha.73

**Comment by slorber** [comment]
This will be fixed in https://github.com/facebook/docusaurus/pull/4654

Also reported here: https://github.com/facebook/docusaurus/issues/4591#issuecomment-822372398

However, can you explain the motivation for having an empty description in that doc?

The goal of this new validation system is to prevent users from doing weird things, and I believe it's a bit weird to use an empty doc description?

**Comment by TomPeirs** [comment]
Thanks for pointing this out.

It's indeed not correct to have empty descriptions indeed. I also just found this as a bug in our repository only after updating.
We basically convert our books from XML to MD via pandoc and I wasn't aware that in a few cases this was empty. But we solved it already on our side. Just wanted to bring it up.

I will try to do a bit more my best to find duplicate issues before reporting them ;-)
I must have overlooked them.
